### DISCUSSION TITLE
[master branch on 1809] I think GlassOverrideAccent is broken (the inactive titlebar is always the same color no matter what)

### DISCUSSION THREAD

**Discussion by Luximoz** [ALTaleX531/OpenGlass#55] [Q&A]
So, I noticed I had my inactive titlebars looking awful where they would have the exact same color as active ones, I tried playing around with GlassOverrideAccent, ColorizationColorBalanceInactive, AccentColorInactive, and ColorizationColorInactive, and no matter what I do, I still get the same color on my inactive titlebars, I've tested this on both 1.2.1 and the new 1.3 version. 

![Registry Editor_w1](https://github.com/user-attachments/assets/ed168384-30af-4ead-98e6-c56377241281)
![Registry Editor_w2](https://github.com/user-attachments/assets/19dcceb7-617a-431c-a749-eec80fc24ac6)

**Comment by ALTaleX531** [comment]
Because `GlassOverrideAccent` is used to replace a buggy effect called **accent**, it is not used to override your accent color.

The usual way to do this is to check the **Title bars and window borders** box in the **Show accent color on the following surfaces** section, then enable `ForceAccentColorization`. Next, you can change the `AccentColorInactive` to adjust the inactive title bar color.

**Reply by Luximoz** [reply]
After some testing around, I found out that I actually need to **disable** "ForceAccentColorization" instead of enabling it like you told me, to avoid having that bug, doesn't matter what the AccentColorInactive value is, if you have ForceAccentColorization on, it will do that bug.

![Registry Editor 1_w2](https://github.com/user-attachments/assets/61a726fd-222a-45c9-b242-01d7e7cbd96e)
![Registry Editor 2_w2](https://github.com/user-attachments/assets/a4158cb5-ab94-4a0e-b9a3-13cbe65fbd8e)

Also, I just noticed that enabling "GlassOverrideAccent" makes Open Shell's start menu glass opacity fully opaque no matter what in case you didn't know about this and if it's an OpenGlass bug and not just Windows 10 being Windows 10.

**Reply by ALTaleX531** [reply]
![image](https://github.com/user-attachments/assets/a581ff03-76fa-4fc6-9023-ea16ffd355c4)
![image](https://github.com/user-attachments/assets/446e4b14-0c4a-4699-91c6-a3fa5ba89ac0)
![image](https://github.com/user-attachments/assets/f8dbf1cf-8403-4959-9fad-7b21f67c9d0d)

I'm not sure what bugs you are referring to, everything works fine here.

**Reply by Luximoz** [reply]
Ooooh, so I just tested this without the uDWM.dll patch (which resizes the titlebar buttons), and now, ForceAccentColorization works as intended. Also in your screenshot, you're using StartIsBack++ for the start menu, I was talking about Open Shell's start menu

**Reply by ALTaleX531** [reply]
<img width="319" alt="image" src="https://github.com/user-attachments/assets/8438f97c-9921-428c-aa93-4921156a1266">
They both use accent so there is basically no difference. Nonetheless, I don't recommend you to use OpenShell.

**Reply by DAPOPOBEFASTONYOAZZ** [reply]
This is old, but I had to comment.

I would similarly advise against OpenShell. It is not as robust as StartIsBack (which I know being paid software is a turn off...)

I would also advise against the uDWM.dll patch, you'll (likely) eventually screw something up and your install is corrupted to the point that even system restore won't save you.

AWM is a much safer alternative. Rather than overriding DWM, it injects its parameters into the DWM for you. It gives much better flexibility and is much safer to use.

Lastly, I recommend using WinPaletter to change your window colors, Both active and inactive. Changing your colors through the registry can be a little weird. Doesn't want to save your settings sometimes.

Edit: If you are wanting the standard difference between active and inactive colors, you should set these colors to the same in WinPaletter. If you're using GlassType 0 (Windows Vista style,) in your registry, you should change the first two hex values in your "AccentColorInactive" from FF to 80. This equals the same type of halving the opacity effect of window border colors that Windows Vista did for inactive window borders. When using GlassType 1 (Windows 7 style,) it is a little trickier. If the Color Balance registry key is lesser than the GlassOpacity registry key, you should leave the first two hex values of the "AccentColorInactive" key as FF. If the inverse is true (Color Balance key value is greater than GlassOpacity) then you should change the first two hex values of the "AccentColorInactive" key from FF to 80.

**Reply by Skyyblaze** [reply]
@DAPOPOBEFASTONYOAZZ Is there any chance that AWM will eventually work on Windows 11 24H2 now that we have OpenGlass and DWMBlurGlass working too?

**Reply by DAPOPOBEFASTONYOAZZ** [reply]
I wish I could say, you may want to direct that question to the developers of AWM. I have not tested AWM on Windows 11, but it sounds like you have and it doesn't.

But to expand on what I was saying for WinPaletter, I'd recommend setting your window colors, both active and inactive, to the same color, but changing your opacity in the registry value. Change the first two hexadecimal values from FF to 80, and you'll get an effect just like Windows Vista and Windows 7.

**Reply by Skyyblaze** [reply]
@DAPOPOBEFASTONYOAZZ I actually didn't try it since I assumed it wouldn't work anyway but maybe I should just test it and see what happens.
